exporter/zipkinexporter/zipkin.go

```diff
@@ -234,190 +151,9 @@ func (ze *zipkinExporter) PushTraceData(ctx context.Context, td consumerdata.Tra
 // on the Zipkin exporter and they have the same logic, then delete all the code
 // below here to allow per-span configuration changes.
 
-const (
```

Comment:
delete (or move if appropriate) the comment above

Reply:
Removed.